doc-content/enterprise-only/admin-and-config/managing-business-central-configuring-githooks-proc.adoc

```diff
@@ -20,33 +20,35 @@ You need to have _cygwin_ installed and configured if you are running {PRODUCT}
 . After successful installation, add the path to _cygwin_ _bin_ folder to your environment PATH variable. E.g. *C:\cygwin64\bin*
```

Comment:
I saw dev add these Windows prereqs and found them hideous but didn't have the wherewithal to propose an alternative. So I suggest now that we remove that Windows section and instead add the following additional prereq under the existing prereqs section as follows:
```
.Prerequsites
* ...
* ...
* (For Windows) You have installed Cygwin with the Git package added during installation, and you have added the path to the Cygwin `/bin` folder to your environment `PATH` variable (for example, `C:\cygwin64\bin`). For Cygwin installation instructions, see http://www.cygwin.com/install.html[Installing and Updating Cygwin Packages].
```

Reply:
Changed this with a few tweaks to the wording.